plugins/aws/fix_plugin_aws/configuration.py

```diff
@@ -268,6 +268,10 @@ class AwsConfig:
         default=True,
         metadata={"description": "Collect resource usage metrics via CloudWatch, enabled by default"},
     )
+    collect_access_edges: Optional[bool] = field(
+        default=False,
```

Comment:
we wanted to make the default true and add a feature flag to disable the functionality until it is working.